src/components/searchBox.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component, Fragment } from "react";
+import InputText from "./inputText";
+
+class SearchBox extends Component {
+    state = {};
+    render() {
+        return (
+            <Fragment>
+                <InputText label="What Pokemon are you looking for?" />
```

Comment:
I understand you don't want to implement the style of the component like the margin or the padding but we can implement libraries like Material-UI and create our own components using the Material-UI components. I think that this component should be a re-usable search bar like the chrome search bar. I wouldn't add the button right. You can check the Autocomplete component of Material-UI for this https://material-ui.com/es/components/autocomplete/